sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_content_downloader_async.py

```diff
@@ -72,7 +72,7 @@ async def download_streaming(  # pylint: disable=inconsistent-return-statements
             params=_params,
             start=offset,
             end=length,
-            host=parsedEndpoint.hostname,
+            host=parsedEndpoint.hostname or "",
```

Comment:
what happens here when you pass in an empty string? or should an error be raise if there is no parsed host